apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -1539,6 +1540,58 @@ static void tc_pthread_pthread_setcanceltype(void)
 }
 #endif
 
+/**
```

Comment:
These descriptions are for `tc_pthread_pthread_mutex_consistent()`, not `test_mutex_consistent()`.
So I think we'd better move these lines to line 1563.